src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const waitForClick = new Promise((resolve, reject) => {
```

Comment:
You don't use `reject` function in the promise
```suggestion
const waitForClick = new Promise((resolve) => {
```